src/services/anchor-service.ts

```diff
@@ -145,38 +145,53 @@ export class AnchorService {
       }
     }
 
-    logger.imp(`Creating Merkle tree from ${candidates.length} selected streams`)
-    const merkleTree = await this._buildMerkleTree(candidates)
+    try {
```

Comment:
Can we put the body of the `try` into a `_anchorCandidates()` helpful function to improve readability?

Reply:
yes will definitely do so